core/frontend/src/EmphasizeElements.ts

```diff
@@ -234,15 +234,34 @@ export class EmphasizeElements implements FeatureOverrideProvider {
     return true;
   }
 
-  /** Clear elements with color/transparency overrides. Specify key to clear only a single override.
+  /** Clear color/transparency overrides from elements. Removes all overrides when neither key or ids is specified.
+   * @params key Specify key value from [[EmphasizeElements.getOverriddenElements]] or [[EmphasizeElements.createOverrideKey]]

```

Comment:
`@param` not `@params`.

Reply:
Wasn't able to build the docs, use to use npx @bentley/bemetalsmith-cli core...but now I just get an error?

Reply:
Not sure if `npm run docs` would flag that (I think not). Send error to Will Bentley? I haven't built the docsite in a while, always have to ask him to remind/update me on the process.